sdk/src/main/java/io/radar/sdk/model/RadarEvent.kt

```diff
@@ -286,27 +306,28 @@ class RadarEvent(
         @JvmStatic
         fun stringForType(type: RadarEventType): String? {
             return when (type) {
+                // These strings should match the values (and order) of the server's event constants.
```

Comment:
have these reorderings already been done in a separate PR? or just on iOS? wondering if a rebase is needed

Reply:
This was done in commit #d5ce87, which was merged into this branch in commit #d9dfbf.